src/process.cpp

```diff
@@ -109,7 +109,7 @@ namespace proc {
   }
 
   boost::filesystem::path
-  find_working_directory(const std::string &cmd, bp::environment &env) {
+  find_working_directory(const std::string &cmd) {
```

Comment:
Can this be merged with the other method with the same name?

Reply:
Can this be moved to the same `user_commands` if something like this gets created?